C/guide/ch_accts.xml

```diff
@@ -45,17 +45,20 @@ Translators:
 
     <figure pgwide="1">
       <title>The basic accounts relationships</title>
+
       <screenshot id="accts-AccountRelationships">
         <mediaobject>
           <imageobject role="html">
             <imagedata fileref="figures/basics_AccountRelationships.png"
                        srccredit="Geert Janssens"/>
           </imageobject>
+
```

Comment:
Jetzt haben wir einen Umbruch zuviel. Da müssen wir wohl woanders noch enen wegnehmen. Low Prio

Reply:
Ist jetzt behoben.

Reply:
Die zusätzliche Leerzeile bekomme ich leider doch nicht weg.
Wenn das Formatierungsprogramm zwei mal gestartet wird, dann ist diese Leerzeile wieder da. Auch wenn es zuvor einmal richtig funktioniert hat.

Reply:
Huch, non-deterministisches Verhalten? Sollten wir im Auge behalten, 3.2.  Line Breaks and Indentation in file:///usr/share/doc/packages/xmlformat/The%20xmlformat%20XML%20Document%20Formatter.html noch mal in Ruhe mit unserer conf abgleichen.

Reply:
Das Dokument habe ich nicht auf der Platte, ich schaue gerade in diese Beschreibung: [https://github.com/someth2say/xmlformat/blob/main/docs.md](https://github.com/someth2say/xmlformat/blob/main/docs.md).

Ich glaube, das Original unter [http://www.kitebird.com/software/xmlformat/](http://www.kitebird.com/software/xmlformat/) gibt es nicht mehr.

Reply:
> Das Dokument habe ich nicht auf der Platte, ich schaue gerade in diese Beschreibung: https://github.com/someth2say/xmlformat/blob/main/docs.md.
> 
> Ich glaube, das Original unter http://www.kitebird.com/software/xmlformat/ gibt es nicht mehr.

Ja, der mögliche Nacholger. Du hast auch https://github.com/someth2say/xmlformat/issues/19 gesehen?